rotkehlchen/tests/db/test_db.py

```diff
@@ -1352,3 +1352,12 @@ def test_ens_mapping(user_data_dir):
         cursor.execute('INSERT INTO ens_mappings(address, ens_name) VALUES(?, ?)', ('0x45E6CA515E840A4e9E02A3062F99216951825eB2', 'LABEL1'))  # noqa: E501
     cursor.execute('INSERT INTO ens_mappings(address, ens_name) VALUES(?, ?)', ('0x4362BBa5a26b07db048Bc2603f843E21Ac22D75E', 'LABEL2'))  # noqa: E501
     assert all(map(lambda x: x is not None, cursor.execute('SELECT last_update FROM ens_mappings').fetchall()))  # noqa: E501
+
+
+def test_database_ignored_assets(user_data_dir):
```

Comment:
no need for this test. Add it in the DB upgrade.

Reply:
okey